packages/yew/src/lib.rs

```diff
@@ -293,21 +293,73 @@ pub mod events {
     };
 }
 
-/// Starts an app mounted to a body of the document.
-pub fn start_app<COMP>()
+use crate::app_handle::AppHandle;
+use web_sys::Element;
+
+/// The main entry point of a Yew application. It works similarly to the `program`
+/// function in Elm. You should provide an initial model, `update` function
+/// which will update the state of the model and a `view` function which
+/// will render the model to a virtual DOM tree. If you would like to pass props,
+/// use the `start_app_with_props` method.
```

Comment:
I don't think the comparison to Elm here is useful for most of our users.
```suggestion
/// The main entry point of a Yew application.
/// If you would like to pass props, use the `start_app_with_props` method.
```

Reply:
To be clear, I don't mind comparisons to Elm, React or whatever, but I think those belong on the website instead of the API reference.

Reply:
I Agree. I didn't write it though, I just copied it from app::App. But nevertheless I will fix it

Reply:
Oh, sorry about that then. I didn't notice